cmd/src/setup.rs

```diff
@@ -136,3 +136,16 @@ pub fn overwrite_config_with_cmd_args(config: &mut TiKvConfig, matches: &ArgMatc
         config.metric.address = metrics_addr.to_owned()
     }
 }
+
+#[allow(dead_code)]
+pub fn validate_config(config: &mut TiKvConfig) {
+    if let Err(e) = check_and_persist_critical_config(config) {
```

Comment:
check_and_persist_critical_config will override last_tikv.toml with current config if there are no errors, it's too early and not suitable to persist current config here because:

1) config.validate() following this line may also occur an error 
2) or users may use `config-check` command to try validating configuration file and doesn't perform a rolling update

Reply:
get, I think checking `last_tikv.toml` is necessary, but we should not rewrite it. I will fix it.